components/user-mgt/org.wso2.carbon.user.mgt/src/main/resources/META-INF/services.xml

```diff
@@ -150,6 +150,21 @@
             </parameter>
         </operation>
 
+        <operation name="listUserByClaim">
+            <parameter name="AuthorizationAction" locked="true">/permission/admin/manage/identity/usermgt/list
+            </parameter>
+        </operation>
+
+        <operation name="bulkImportUsers">
+            <parameter name="AuthorizationAction" locked="true">/permission/admin/manage/identity/usermgt/create
+            </parameter>
+        </operation>
+
+        <operation name="getAllSharedRoleNames">
+            <parameter name="AuthorizationAction" locked="true">/permission/admin/manage/identity/usermgt/view
```

Comment:
shouldn't this be rolemgt/view?

Reply:
Fixed in 29d26c733327aa66bf8894e97351ffe0e8346663